Source/InMobiAdapter.swift

```diff
@@ -65,8 +66,13 @@ final class InMobiAdapter: NSObject, PartnerAdapter {
     /// - parameter request: Information about the ad load request.
     /// - parameter completion: Closure to be performed with the fetched info.
     func fetchBidderInformation(request: PreBidRequest, completion: @escaping ([String : String]?) -> Void) {
-        // InMobi does not currently provide any bidding token
-        completion(nil)
+        log(.fetchBidderInfoStarted(request))
+        guard let bidToken = IMSdk.getToken() else {
+            log(.fetchBidderInfoFailed(request, error: error(.prebidFailureUnknown, description: "Failed to provide bid token.")))
+            return
```

Comment:
Let's call `completion(nil)` here to make the operation as finished.

Reply:
https://github.com/ChartBoost/chartboost-mediation-ios-adapter-inmobi/pull/59/commits/401336ce21222c11b52734a3d085b01a7dcc58aa